### PR TITLE
Add provisional product specification for burnt area rasters.

### PR DESCRIPTION
**Pull request by james-blitzm**

### CODE REVIEW

**Comment by james-blitzm** [comment]
@jeremyh @uchchwhash It looks like the above CI failures already exist in the project and aren't related to my changes here?

**Comment by uchchwhash** [comment]
@james-blitzm that is correct!

**Review by uchchwhash — APPROVED** [review]
LGTM!

**Review by jeremyh — COMMENTED** [review]
Thanks James. I've been meaning to add [eo3-validate](https://github.com/GeoscienceAustralia/eo-datasets#validator) to our CI checks -- it's reporting a schema error:
```
❯ eo3-validate 'https://raw.githubusercontent.com/GeoscienceAustralia/digitalearthau/4d631b8e1524812417b030ac2a15eb94fe9b8105/digitalearthau/config/eo3/products-aws/ga_s2_ba_provisional_3.yaml'
0 ODC products
✗ https://raw.githubusercontent.com/GeoscienceAustralia/digitalearthau/4d631b8e1524812417b030ac2a15eb94fe9b8105/digitalearthau/config/eo3/products-aws/ga_s2_ba_provisional_3.yaml
	W missing_suffix Document looks like a product but does not have filename extension ".odc-product.yaml"
	E document_schema (measurements.3.units) 1 is not of type 'string' 

failure: 1 warning, 1 error
```

Ideally we want to run the validator with a dataset too to make sure they match each-other -- is there an example dataset somewhere?

**Comment by jeremyh** [comment]
The CI was fixed in the upstream branch, so the error messages are relevant now.

File: digitalearthau/config/eo3/products-aws/ga_s2_ba_provisional_3..odc-product.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+metadata_type: eo3
+description: Sentinel 2 Burnt Area Collection 3 (Provisional)
+
+license: CC-BY-4.0
+
+metadata:
+  product:
+    name: ga_s2_ba_provisional_3
+  properties:
+    eo:platform: sentinel-2
```
Comment (uchchwhash) [inline digitalearthau/config/eo3/products-aws/ga_s2_ba_provisional_3..odc-product.yaml]: this line will have to be removed now

Comment (james-blitzm) [reply digitalearthau/config/eo3/products-aws/ga_s2_ba_provisional_3..odc-product.yaml]: I've removed this field.

**Comment by james-blitzm** [comment]
Some recent changes:
1. Renamed the file as it was wrong.
2. Removed the `platform` field
3. Updated the measurement names to match the generated outputs (from alchemist).

I have been able to successfully index burn rasters using this specification in my dev environment.

**Comment by jeremyh** [comment]
Could you put one of your output dataset documents into the `digitalearthau/config/eo3/products-aws/examples` folder? 

That way the linter will correctly check that measurements+metadata line up with it, etc. And it's a useful reference to have.

**Comment by james-blitzm** [comment]
> Could you put one of your output dataset documents into the `digitalearthau/config/eo3/products-aws/examples` folder?
> 
> That way the linter will correctly check that measurements+metadata line up with it, etc. And it's a useful reference to have.

Added in latest commit -  https://github.com/GeoscienceAustralia/digitalearthau/pull/304/commits/381b1ea636677415ea2e31f4870fdc448570e093

**Pull request merged**